bugbug/models/fenixcomponent.py

```diff
@@ -85,11 +85,14 @@ def get_labels(self):
         date_limit = datetime.now(timezone.utc) - relativedelta(years=2)
 
         for bug_data in bugzilla.get_bugs():
-            if bug_data["product"] != "Fenix":
+            if bug_data["product"] != "Fenix" and bug_data["product"] != "GeckoView":
```

Comment:
nit: Adding a comment explaining why we are including `GeckoView` would be helpful for future maintainers (including us).

Reply:
Done here: 082ac3d0611ff5d2eac2c597fe3894fac5d78c7e